internal/topology/variables/schema.go

```diff
@@ -166,6 +166,58 @@ func convertToAPIExtensionsJSONSchemaProps(schema *clusterv1.JSONSchemaProps, fl
 		props.XValidations = convertToAPIExtensionsXValidations(schema.XValidations)
 	}
 
```

Comment:
nit: let's move this above the XValidations (mostly just for a bit of consistency to have it closer to Items etc.)

Reply:
Done.